src/usePresence.tsx

```diff
@@ -44,16 +44,16 @@ export function usePresence<T extends any>(
     presence.current?.set(value, expiresAfter);
   }, []);
 
-  return [
-    val,
-    {
-      set: bufferedSet,
-      getMine: () => {
-        return presence.current?.getMine();
-      },
-      getAll: () => {
-        return presence.current?.getAll() ?? {};
-      },
+  //  useState here so object stays the same, preventing spurious re-renders
+  const [wrapper] = useState<PresenceClient<T>>({
```

Comment:
This is better accomplished with `useRef` instead, since the `useState` would encourage another programmer to change this via a rerender.